cmd/program/program.go

```diff
@@ -1,3 +1,5 @@
+// Package program contains utility that help
```

Comment:
Nit: Its not necessarily utility, program is kind of the back bone of the entire logic for the CLI

Reply:
Yeah that was definitely a late night moment. I'll rewrite it quick.